src/main/kotlin/racing/domain/strategy/RandomNumberGenerator.kt

```diff
@@ -0,0 +1,9 @@
+package racing.domain.strategy
+
+import java.util.concurrent.ThreadLocalRandom
+
+object RandomNumberGenerator : Generator<Int> {
+    private const val THRESHOLD = 10
+
+    override fun generate(): Int = ThreadLocalRandom.current().nextInt(THRESHOLD)
```

Comment:
코틀린의 Random은 기본적으로 ThreadSafe합니다! [문서](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.random/-random/-default/)

또한 Collection에서도 `random()` 함수를 지원하는데요, 사소하지만 다음과 같이 표현할 수 도 있을 것 같아요!
```suggestion
    override fun generate(): Int = (0..9).random()
```

Reply:
네 확인해보니 코틀린에서는 jdk 8부터 Random에서도 자체적으로 스레드 안전하게 지원한다고 하네요 ㅎㅎ 감사합니다